### ISSUE TITLE
gminer issue

### ISSUE THREAD

**Issue by MaynardMiner** [MaynardMiner/SWARM#31]
I have heard multiple reports about gminer not working on linux side, reports as far as reason why is not know as of yet. I am investigating. Before release it benchmarked for me.

**Comment by MaynardMiner** [comment]
```
[0m[0;31m00:48:53 Anti-hacking system detected modification of the miner memory. If you using an antivirus, add miner to antivirus exceptions
[0m[0;31m00:48:53 Miner terminated, watchdog will restart process after 10 seconds
[0mterminate called without an active exception
[0;31m00:49:03 Watchdog unable to start miner
```

I believe this may be related to the fact SWARM is attempting to | tee miner output to a logfile.

I am going to attempt to use his logfile command instead.

**Comment by MaynardMiner** [comment]
Proposed fix applied: 2.0.6
